docker/ci/Dockerfile

```diff
@@ -38,9 +38,10 @@ RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8
 
 RUN npm install -g bower polymer-bundler
 
-# RUN curl -s https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
-#     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
-#     apt-get update && apt-get install -y google-chrome-stable
+# Install latest Chrome stable, needed for chromedriver testing.
```

Comment:
This move is to try reenabling js_unittest next in CI. we need to have google chrome stable there first.